### PR TITLE
SOCRadar v1.1

### PR DESCRIPTION
**Pull request by semsaksoy**

### CODE REVIEW

File: socradar/info.json [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+        "value": "https://platform.socradar.com/api"
+      },
+      {
+        "title": "Verify SSL",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "value": "true",
+        "name": "verify"
```
Comment (cs-renuka-andhale) [inline socradar/info.json:34]: Please provide description:
Specifies whether the SSL certificate for the server is to be verified. By default, this option is set to True.

File: socradar/info.json [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+      "annotation": "get_incident",
+      "parameters": [
+        {
+          "title": "Alarm ID",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "integer",
+          "name": "alarm_id",
+          "description": ""
```
Comment (cs-renuka-andhale) [inline socradar/info.json]: Please provide description.

File: socradar/info.json [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+    },
+    {
+      "operation": "threat_analysis",
+      "title": "Threat Analysis",
+      "description": "You can obtain the 'trust' score (out of 100) of an entity that is calculated based on various threat feed evaluations conducted by SOCRadar",
+      "category": "investigation",
+      "annotation": "threat_analysis",
+      "parameters": [
+        {
+          "title": "entity",
```
Comment (cs-renuka-andhale) [inline socradar/info.json]: Entity

File: socradar/info.json [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+      "enabled": true,
+      "output_schema": {
+        "data": "",
+        "is_success": "",
+        "message": "",
+        "response_code": ""
+      }
+    }
+  ],
+  "playbooks": [
```
Comment (cs-renuka-andhale) [inline socradar/info.json]: this key "playbooks" can be removed.

File: socradar/info.json [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+    {}
+  ],
+  "icon_small": " data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAMAAABEpIrGAAAAhFBMVEX////a2+JlZYOXmKvU1NxzdI+Cgpp7fJXg4efNztjr7PBub4rz9Pbl5uv/r7uRkab7/P2NjKKur76IiJ92eJGjna/+w8z/5ur/naz4obD4+fqpprj/orH/o7H+8vS+v8vqw83Vvcn/1Nv/o7LYo7PRobHYsb/8ucS4ucbHvsr/24PaZasGmS6PAAAAu0lEQVQ4y+1SNxLCMBBUPp2CJVs2GAdMNvz/g8RhKDAdVGxzxe7MhjlC/vgh5rMsmy2n+U297fuunr/jmoY4cRgX+90iW6F2VPg7Ye43IpIE6ZRDWa0lQMGURSoaJzj3aAylVBNVHo/gtJS8McgG23JZBWB5oSCwIhIiCluqmyBeBKUckrKMVioEBvxqk5JtnxYgIECucrAguVXtLQ6+hNRReyG8Q+o1Um2eNbupmo+hxs9DXfv+3+n7OAN6qgvGvSYIUQAAAABJRU5ErkJggg==",
```
Comment (cs-renuka-andhale) [inline socradar/info.json]: please remove "icon_small".

File: socradar/info.json [hunk 6]
```diff
@@ -0,0 +1,402 @@
...
...7bHutvu4VW89XZzyPMTUt4vm3X9sXgRcVlt181hO1De7Np89xSiY6fmp5ov2rz+qPdIeb3YVPtmt3sC0M4CCxEhzaKVRdfbY2OjFtK27k5QpINOIAxQEtv5eTujoyoJgyy1ibMSWmrF7ZxGn9py13rlWU+poVGU8M0RoCvZ+ChNwsQkltJG5AuVgKs1M2IlY4+YCMM46oZnSMF5rLULhJpeMUTC8Tw6AEKBGvpGKZiec45hpJWF3mYOIvseLQIhfcfVyjWdIuEJNSSnjhfL2Aw4E7GLV9KSoTXJCtuPBeMWSQ0/YiI7+xmgh4EJBHxpFP3BSouQIUqvF8QxZAnE4eg3F+i/Lh2bSSUwDqwIVGwIKVJLK6VEcgYsfCUV8210JTpHWygM5uozLngcZ5mrkDISMtGa7UuR2ZlAD6nlmSmTUR+2MP3KEQ04JoBZX4DeGFArzIpvQugbvFQqiaU/EGKFueJGyaSfKB18J9sh1F4JWZbSGGxkWlhflGlPOQ64FimxhDLKWKuADZYjyWyuWWmoHlCF/wCtMnSIGUuOwcRE0ITJsk9KhLrFLfSwxChgoso+xqFSSMUwsQ4kC/CBUD4CjLEsXK6k6/Rlw9EgswkWi457vS8bDlhRIvRlnHq+VmmmteH2dYWAKy8RYw8RTEpVek3hYiwlFjM2oleqXjdCOmTZY7q0mBaCCZVSF2+oWDHPx9wKpq8BSWFFbtS33r6xV6gVy6H1QtSTgoCVhdDXaYKRdle2FWAp4Y3ECxKHBq5rpVlqBsGoa9AnHIj1aZnn7aLaPDkcrsdXmx+qbrOtuq5um93LA/FUbY44YBe73bEdBuzNxQIPHra4Ag7rnvK+7qoj/MmS6rqqqi6WFEz8zn/Tp2NHm/aIlA+nfHbXPnI4/9Fz7OEeGvyj+k3v8uzq/V0jcI/O1EW4ivZrBG9+MOJ0bfvy5AcoPGP1Tmy+DcKNKzATTyAP+Qdzb+F+7uAeradLcswQJl6PS+TCa3jQhZ+w8Hj2YJoo/C7vtwsL7vT2VEsA+UWnXdY13IvNHZynexEmO+0We5jM/++TRh4YWvNBfGiQzaQDJpMI5CbnCcfv8Ybnhu7DX8PNgp54hh/hgJ8tfGvswQ0b08BPjR74/Y6cJz5RxTC3vKd4wCM9B8/1wdxyeXmtwmwDjIbPf+scQYXIC+DQAAAAAElFTkSuQmCC"
```
Comment (cs-renuka-andhale) [inline socradar/info.json]: please remove "icon_large".

File: socradar/operations.py [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+            raise ConnectorError("Unable to connect SOCRadar")
+        return True
+    except Exception as e:
+        raise ConnectorError("Unable to connect SOCRadar {0}".format(e))
+
+
+operations = {"get_incidents": get_incidents, "get_incident": get_incident, "threat_analysis": threat_analysis,
+              "change_status": change_status}
+
+status_map = {"OPEN": 0,
```
Comment (cs-renuka-andhale) [inline socradar/operations.py]: Please move this "status_map" to top.

**Comment by semsaksoy** [comment]
done

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**